<html>
<head>
    <style>
        /* hides "Team:Cornell" and iGEM logo */
        #top_title {
            display: none;
        }

        /* removes padding below footer */
        #globalWrapper {
            padding: 0;
        }

        #content {
            background: #0d1c38;
            padding: 0px;
            width: 100% !important;
            margin-top: 0px;
            margin-left: 0px ;
        }

        /* removes default styling for home banner tagline */
        #HQ_page p {
            font-family: 'Open Sans', sans-serif;
            font-size: 16px;
        }

        /* removes small extra margin at very bottom below footer */
        p {
            margin: 0;
        }

        /* removes bullets from Toolkit dropdown menu */
        ul {
            list-style-image: none;
            list-style-type: disc;
        }
    </style>
	<title>Team:Cornell/Practices - 2018.igem.org</title>
	<!-- CSS -->
    <link rel="stylesheet" type="text/css" href="http://2018.igem.org/Team:Cornell/styles?action=raw&ctype=text/css">
    <link rel="stylesheet" type="text/css" href="http://2018.igem.org/Team:Cornell/grids?action=raw&ctype=text/css">
    <!-- JS -->
    <script src="http://2018.igem.org/Team:Cornell/jquerymin?action=raw&ctype=text/javascript"></script></script>
    <script src="http://2018.igem.org/Team:Cornell/mainJS?action=raw&ctype=text/javascript"></script></script>
</head>
<body>
    <div class="practices-page-wrapper">
        <!------------------------ NAV BAR START ------------------------>
        <nav>
            <div class="nav-bar-wrapper">
                <div>
                    <div class="nav-bar-logo">
                        <a href="http://2018.igem.org/Team:Cornell"><img src="http://2018.igem.org/wiki/images/6/63/T--Cornell--OscillateLogo.jpg" alt="Oscillate"></a>
                    </div>
                    <div class="nav-bar-main-menu">
                    <div class="dropdown">
                        <button class="dropbtn">TEAM</button>
                        <div class="dropdown-content team-dropdown-content">
                            <a href="http://2018.igem.org/Team:Cornell/Team">BIOS</a>
                            <a href="http://2018.igem.org/Team:Cornell/Sponsors">SPONSORS</a>
                            <a href="http://2018.igem.org/Team:Cornell/Attributions">ATTRIBUTIONS</a>
                        </div>
                    </div>
                    <div class="dropdown">
                        <button class="dropbtn">OUTREACH</button>
                        <div class="dropdown-content outreach-dropdown-content">
                            <a href="http://2018.igem.org/Team:Cornell/Collaborations">COLLABORATIONS</a>
                            <a href="http://2018.igem.org/Team:Cornell/Public_Engagement">PUBLIC ENGAGEMENT</a>
                        </div>
                    </div>
                    <div class="dropdown">
                        <button class="dropbtn active-page">HUMAN-CENTERED DESIGN</button>
                        <div class="dropdown-content human-dropdown-content">
                            <a href="http://2018.igem.org/Team:Cornell/Human_Practices">PRACTICES</a>
                            <a href="http://2018.igem.org/Team:Cornell/Policies">POLICIES</a>
                            <a href="http://2018.igem.org/Team:Cornell/Entrepreneurship">ENTREPRENEURSHIP</a>
                        </div>
                    </div>
                    <div class="dropdown">
                        <button class="dropbtn">MODELING</button>
                        <div class="dropdown-content modeling-dropdown-content">
                            <a href="http://2018.igem.org/Team:Cornell/Modeling">MODELING</a>
                            <a href="http://2018.igem.org/Team:Cornell/Testing">TESTING</a>
                        </div>
                    </div>
                    <div class="dropdown">
                        <button class="dropbtn">TOOLKIT</button>
                        <div class="dropdown-content toolkit-dropdown-content">
                            <ul>
                                <div class="nav-first-col">
                                    <li class = "wet-lab-list-title"><b>WET LAB</b></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/Foundations">FOUNDATIONS</a></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/Demonstrate">DEMONSTRATE</a></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/InterLab">INTERLAB</a></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/Parts">PARTS</a></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/Basic_Part">BASIC PARTS</a></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/Composite_Part">COMPOSITE PARTS</a></li>
                                    </li>
                                </div>
                                <div class="nav-second-col">
                                    <li class = "doc-list-title"><b>DOCUMENTATION</b></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/Notebook">NOTEBOOK</a></li>
                                    <li><a href="http://2018.igem.org/Team:Cornell/Safety">SAFETY</a></li>
                                    </li>
                                </div>
                            </ul>
                        </div>
                    </div>
                    <div class="dropdown">
                        <button class="dropbtn" onclick="location.href='http://2018.igem.org/Team:Cornell/About'">ABOUT</button>
                    </div>
                    <div class="dropdown">
                        <button class="dropbtn" onclick="location.href='http://2018.igem.org/Team:Cornell'">HOME</button>
                    </div>
                </div>
                </div>
            </div>
        </nav>
        <!------------------------ NAV BAR END ------------------------>
        
        <!------------------------ STANDARD PAGE BANNER START ------------------------>
        <header class="standard-page-banner practices-page-banner">           
                <svg viewBox="0 0 100 100" width=100% height=100%>
                    <polygon points="-50,50 50,10 150,50 50,90" fill="white" fill-opacity="0.4"></polygon>
                    <text text-anchor="middle" alignment-baseline="middle" x=50% y=50%>Practices</text>
                </svg>
        </header>
        <!------------------------ STANDARD PAGE BANNER END ------------------------>

        <!------------------------ PRACTICES PAGE CONTENT START ------------------------>
        <div class="practices-page-content-wrapper">
            <!------------------------ PRACTICES OVERVIEW START ------------------------>
            <div class="practices-overview-wrapper">
                <div class="standard-page-content-title">Overview</div>
                <hr class="yellow-accent-line-left">
                <div class="standard-page-content-subheading"></div>
                <div class="practices-overview">
                    <p>It started with an idea.</p><br>

                    <p>A frequency-based band pass filter.</p><br>

                    <p>This year, we have taken on a project like none we have done before. Oscillate aims to break through existing biotechnologies through our foundational advance in frequency rather than amplitude sensitive biological systems. Our multifaceted approach to tackling this project spanned research, interacting with experts, and exploring applications. We started with the basics and went to the foundations of synthetic biology. We spoke with professors in fields ranging from electrical and computer engineering to biomedical engineering and gauged the feasibility of our ideas. We listened to their feedback and ideas and incorporated them into our wet lab practices and modeling. After implementing their considerations into our design, we also tested our project with expert feedback and advice in mind. Finally, we went to various fields to discuss the applications and implications of our project, doing all this while considering the ethical and entrepreneurial implications of our project.</p><br>

                    <p>This year, we have taken on a project like none we have done before. Oscillate aims to break through existing biotechnologies through our foundational advance in frequency rather than amplitude sensitive biological systems. Our multifaceted approach to tackling this project spanned research, interacting with experts, and exploring applications. We started with the basics and went to the foundations of synthetic biology. We spoke with professors in fields ranging from electrical and computer engineering to biomedical engineering and gauged the feasibility of our ideas. We listened to their feedback and ideas and incorporated them into our wet lab practices and modeling. After implementing their considerations into our design, we also tested our project with expert feedback and advice in mind. Finally, we went to various fields to discuss the applications and implications of our project, doing all this while considering the ethical and entrepreneurial implications of our project.</p><br>

                    <p>We present our work as it relates to four themes: Project Conception, Evaluation Biological Components, Optimization and Modeling, Testing. For each of these themes, we <b><i>recognized</i></b> our project’s limitations, <b><i>explored</i></b> different aspects of our questions through interviews, and <b><i>implemented</i></b> our findings into our project.</p><br>

                    <p>We <b><i>integrated</i></b> what we learned through our human practices and <b><i>delved</i></b> into the vast possibilities of applications.</p>
                </div>
            </div>
            <!------------------------ PRACTICES OVERVIEW END ------------------------>
            <!------------------------ PRACTICES BUTTONS START ------------------------>
            <div class="practices-page-buttons-wrapper">
                <div class="practices-button-wrapper">
                    <button class="practices-button" id="showAll">VIEW ALL</button>
                </div>
                <div class="practices-button-wrapper">
                    <button class="practices-button button-single" id="1">CONCEPTION OF THE PROJECT</button>
                </div>
                <div class="practices-button-wrapper">
                    <button class="practices-button button-single" id="2">CHOOSING BIOLOGICAL COMPONENTS</button>
                </div>
                <div class="practices-button-wrapper">
                    <button class="practices-button button-single" id="3">OPTIMIZATION AND MODELING</button>
                </div>
                <div class="practices-button-wrapper">
                    <button class="practices-button button-single" id="4">TESTING</button>
                </div>
                <div class="practices-button-wrapper">
                    <button class="practices-button button-single" id="5">UNDERSTANDING APPLICATIONS</button>
                </div>
            </div>
            <!------------------------ PRACTICES BUTTONS END ------------------------>
            <div class="practices-page-content-section">
                <!------------------------ PRACTICES PROJECT CONCEPTION ------------------------>
                <div class="practices-text-wrapper" id="div1">
                    <div class="practices-recognition-wrapper">
                        <div class="standard-page-content-title">Recognition</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-recognition">
                            <p>As we took off with our idea of a frequency-sensitive genetic circuit, we knew that implementing this abstract idea could not be done solely within the confines of our iGEM team. Therefore, we drew inspiration from a whole host of experts in fields such as chemical engineering, bioengineering, biophysics, and computational biology. Throughout every stage of our project, we asked critical questions, for example, on the assumptions we held about our project, always seeking to involve the academic community in the conversation when bringing our project to fruition. Follow us on this page through our design process for Oscillate!</p>
                        </div>
                    </div>
                    <div class="practices-exploration-wrapper">
                        <div class="standard-page-content-title">Exploration</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-exploration-section-left">
                            <div class="practices-exploration-text-left">
                                <div class="practices-page-content-subheading">
                                    Prof. Guillaume Lambert, School of Applied and Engineering Physics, Cornell University
                                    <div class="practices-explore-date">April 9, 2018</div>
                                </div> 
                                <p>Professor Lambert helped us narrow down the choice of stimuli for our band-pass filter, reminding us of the <b>different qualities of stimuli</b> such as <b>temperature</b>, <b>light</b>, or <b>small molecules</b>. Our conversations ranged from testing methodologies to evaluating possible candidates for the gene components of our project. Through it all, he encouraged us to keep the circuit simple so it can be better modeled and its individual components tested down the line.</p>
                            </div>
                            <div class="practices-exploration-image-right">
                                <img src="http://2018.igem.org/wiki/images/4/41/T--Cornell--Lambert.jpg">
                            </div>
                        </div>

                        <div class="practices-exploration-section-right">
                            <div class="practices-exploration-image-left">
                                <img src="http://2018.igem.org/wiki/images/2/22/T--Cornell--JohnMarch.jpg">
                            </div>
                            <div class="practices-exploration-text-right">
                                <div class="practices-page-content-subheading">
                                    Prof. John March, School of Biological and Environmental Engineering, Cornell University 
                                    <div class="practices-explore-date">April 21, 2018</div>
                                </div> 
                                <p>Professor March’s experience with engineering cell signaling pathways helped us frame our project in terms of <b>canonical biological circuits</b>. He introduced us to the potential of developing the <b>feedforward loop motif</b> that was necessary in our circuit.</p>
                            </div>
                        </div>

                        <div class="practices-exploration-section-left">
                            <div class="practices-exploration-text-left">
                                <div class="practices-page-content-subheading">
                                    Prof. Edwin Kan, School of Electrical and Computer Engineering, Cornell University 
                                    <div class="practices-explore-date">April 10, 2018</div>
                                </div> 
                                <p>Professor Kan specializes in biosensors and electrical systems that interface with physiological systems. In our conversation, he provided us with guiding principles to inform the design of our project. He tasked us with considerations such as the <b>weighted logic of bacterial</b> compared to the digital logic of electronics, the presence of <b>biological noise</b>, and the ever present issues of <b>lag time</b> in the response of a possible sensor. These were all important questions for us to ask ourselves as we worked on creating the biological equivalent of a band-pass filter in bacteria.</p>
                            </div>
                            <div class="practices-exploration-image-right">
                                <img src="http://2018.igem.org/wiki/images/f/f1/T--Cornell--Kan.jpg">
                            </div>
                        </div>
                        
                        <div class="practices-exploration-section-right">
                            <div class="practices-exploration-image-left">
                                <img src="http://2018.igem.org/wiki/images/7/71/T--Cornell--Nika.jpg">
                            </div>
                            <div class="practices-exploration-text-right">
                                <div class="practices-page-content-subheading">
                                    Dr. Nika Shakiba, Department of Biological Engineering, Weiss Lab for Synthetic Biology, Massachusetts Institute of Technology 
                                    <div class="practices-explore-date">August 6, 2018</div>
                                </div> 
                                <p>Dr. Shakiba is a postdoctoral fellow at the Weiss Lab for Synthetic Biology at MIT, a lab at the edge of innovation in synthetic biology. Dr. Shakiba’s holistic input on the general nature of our circuit, which appeared to her as an incoherent feedforward loop, took into account the <b>tuning</b> of the two components of the circuit, the <b>activity of the promoter</b>, and the <b>rates of degradation</b> of the projects. Our discussion with her served as a good checkpoint for refining our project.</p>
                            </div>
                        </div>
                    </div>
                    <div class="practices-implementation-wrapper">
                        <div class="standard-page-content-title">Implementation</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <ul>
                            <li>We decided to use <b>temperature</b> response for our band-pass filter over <b>light</b> and <b>small molecules</b>, due to the abundance of testing methods and less potential error for testing our construct.</li>
                            <li>We developed a <b>feedforward loop</b> for our circuit and framed our project in terms of <b>canonical biological circuits</b>.</li>
                            <li>We considered the <b>weighted logic of bacterial</b> compared to the digital logic of electronics, the presence of <b>biological noise</b>, and the issues of <b>lag time</b> in the response of our sensor.</li>
                            <li>It is important to address the <b>tuning</b> of the two components of the circuit, the <b>activity of the promoter</b>, and the <b>rates of degradation</b> of the components.</li>
                        </ul>
                    </div>
                </div>
                <!------------------------ PRACTICES BIOLOGICAL COMPONENTS ------------------------>
                <div class="practices-text-wrapper" id="div2">
                    <div class="practices-recognition-wrapper">
                        <div class="standard-page-content-title">Recognition</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-recognition">
                            <p>At the beginning of our project, we had to narrow down the pool of gene candidates for each component of our biological circuit. This meant literature searches and prolonged discussions on potential sequences for use. In order to optimize our circuit, we turned to experts in diverse fields right at Cornell.</p>
                        </div>
                    </div>
                    <div class="practices-exploration-wrapper">
                        <div class="standard-page-content-title">Exploration</div>
                            <hr class="yellow-accent-line-left">
                            <div class="standard-page-content-subheading"></div>
                            <div class="practices-exploration-section-left">
                                <div class="practices-exploration-text-left">
                                    <div class="practices-page-content-subheading">
                                        Prof. Guillaume Lambert, School of Applied and Engineering Physics, Cornell University
                                        <div class="practices-explore-date">April 9, 2018</div>
                                    </div> 
                                    <p>Professor Lambert specializes in biological microfluidics and the development and characterization of synthetic biological regulatory elements for the purpose of creating biological circuits. He first shared his concerns regarding lag time in receiving the frequency and with reporting the output. To solve this, he mentioned the possibility of <b>tuning the system</b>, with <b>different degradation tags</b> and using the <b>orthogonal mf-Lon system</b>. These suggestions had a major influence on our modeling and design process. We discussed the possibility of other reporters that could be used to screen the effectiveness of our system, such as antibiotic resistance genes that would be activated by our system. This reminded us to not simply assume fluorescence is the best reporter for our project. In our recurring contact with him, we continually addressed questions not only in our design, but also in the eventual testing of our circuit.</p>
                                </div>
                                <div class="practices-exploration-image-right">
                                    <img src="http://2018.igem.org/wiki/images/4/41/T--Cornell--Lambert.jpg">
                                </div>
                            </div>

                            <div class="practices-exploration-section-right">
                                <div class="practices-exploration-image-left">
                                    <img src="http://2018.igem.org/wiki/images/2/22/T--Cornell--JohnMarch.jpg">
                                </div>
                                <div class="practices-exploration-text-right">
                                    <div class="practices-page-content-subheading">
                                        Prof. John March, School of Biological and Environmental Engineering, Cornell University 
                                        <div class="practices-explore-date">April 21, 2018</div>
                                    </div> 
                                    <p>Professor March was one of the first experts we contacted to evaluate our biological band-pass filter based on his comprehensive knowledge of rewiring cellular signaling circuitry. In evaluating our system, he suggested approaching the problem of lingering protein intermediates and products in the system by using plasmids with different copy numbers to tune the relative levels of each type of protein. At this stage, we had not yet decided on how we would approach the temperature-sensitive activation of our band-pass filter. Professor March helped <b>begin our search</b> by providing us with resources on <b>heat-sensitive promoters</b> and <b>heat-shock proteins</b>.</p>
                                </div>
                            </div>

                            <div class="practices-exploration-section-left">
                                <div class="practices-exploration-text-left">
                                    <div class="practices-page-content-subheading">
                                        Prof. Matthew DeLisa, School of Chemical and Biomolecular Engineering, Cornell University 
                                        <div class="practices-explore-date">July 9, 2018</div>
                                    </div> 
                                    <p>We consulted Professor DeLisa on every part of our system. In regards to our protein degradation tags, he voiced his concern of <b>not having such tags on HrpR and HrpS</b> and the <b>problems</b> with having these <b>proteins lingering in cell</b>. He also echoed Professor Lambert’s suggestion of using the mfLon-ssrA orthogonal protease system for protein degradation. While he could not provide feedback on the HrpLRS system in detail, he did point us to Professor Swingle and his expertise. Finally, he brought up the possibility of using RNA aptamers due to their fast generation and degradation rates. The aptamers would fluoresce in the presence of certain small molecules in the solution as the reporter for this system and an alternative to GFP.</p>
                                </div>
                                <div class="practices-exploration-image-right">
                                    <img src="http://2018.igem.org/wiki/images/3/3e/T--Cornell--DeLisa.jpg">
                                </div>
                            </div>
                            
                            <div class="practices-exploration-section-right">
                                <div class="practices-exploration-image-left">
                                    <img src="http://2018.igem.org/wiki/images/9/93/T--Cornell--Swingle.jpg">
                                </div>
                                <div class="practices-exploration-text-right">
                                    <div class="practices-page-content-subheading">
                                        Prof. Bryan Swingle, School of Integrative Plant Science, Cornell University
                                        <div class="practices-explore-date">August 6, 2018</div>
                                    </div> 
                                    <p>Professor Swingle specializes in understanding the molecular systems that enable bacteria to adjust and thrive in plants and other environments. The HrpLRS system originated from the plant pathogen Pseudomonas syringae and is a crucial component of our system as an AND gate for transcription. Thus, we met with Professor Swingle to discuss the dynamics of this molecular system, giving careful consideration to the kinetics of HrpR and HrpS dimerization and the rate at which this dimer activates transcription via the HrpL promoter. This was to address a recurring concern that our system would suffer from lag time between its components. From his experience, Professor Swingle was able to say that this <b>system displays fast kinetics</b> but did not have quantitative numbers for support. He pointed us to testing methodologies in order to provide realistic parameters for our model of the system and to rule out significant lag time problems from this component of our system.</p>
                                </div>
                            </div>
                        </div>
                        <div class="practices-implementation-wrapper">
                            <div class="standard-page-content-title">Implementation</div>
                            <hr class="yellow-accent-line-left">
                            <div class="standard-page-content-subheading"></div>
                            <ul>
                                <li>Our system was designed to be tuned using <b>different degradation tags</b> and the <b>orthogonal mf-Lon system</b>.</li>
                                <li>Thanks to our introduction to <b>heat-sensitive promoters</b> and <b>heat-shock proteins</b>, we decided to use a <b>RNA thermometer</b> for our temperature dependent component.</li>
                                <li>We modeled <b>protein degradation tags on HrpR and HrpS<b>.</li>
                                <li>The use of HrpR and HrpS allowed our system to <b>display fast kinetics</b> due to the proteins’ fast dimerization kinetics.</li>
                            </ul>
                        </div>
                    </div>
                    <!------------------------ PRACTICES OPTIMIZATION AND MODELING ------------------------>
                    <div class="practices-text-wrapper" id="div3">
                        <div class="practices-recognition-wrapper">
                        <div class="standard-page-content-title">Recognition</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-recognition">
                            <p>Our bio-circuit takes the input of an oscillating temperature and, with a low- and high-pass filter, can ultimately lead to the frequency-dependent expression of GFP. Due to the complexity of our mechanism of action, we needed a way to characterize the relationship between our inputs and outputs. Therefore, we developed both a deterministic and stochastic model to better understand our system. Through our interviews with researchers who are developing new ways to better model biological pathways, we aimed to learn how to improve our modeling approach.</p>
                        </div>
                    </div>
                    <div class="practices-exploration-wrapper">
                        <div class="standard-page-content-title">Exploration</div>
                            <hr class="yellow-accent-line-left">
                            <div class="standard-page-content-subheading"></div>
                            <div class="practices-exploration-section-left">
                                <div class="practices-exploration-text-left">
                                    <div class="practices-page-content-subheading">
                                        Prof. Guillaume Lambert, School of Applied and Engineering Physics, Cornell University
                                        <div class="practices-explore-date">July 10, 2018</div>
                                    </div> 
                                    <p>Early on in the development of our model, Professor Lambert provided us with advice on how to make our model more representative of our system. His research focuses in part on the development of synthetic biological regulatory elements and the creation of biological circuits, so his insight into what factors were essential to describe our system was invaluable. He emphasized <b>considering both the cell growth rate and protein degradation rate</b>, because both could be characterized by measuring our degradation rate with and without tags. Furthermore, he pointed us to research that would help determine the rate based on our specific degradation machinery.</p>
                                </div>
                                <div class="practices-exploration-image-right">
                                    <img src="http://2018.igem.org/wiki/images/4/41/T--Cornell--Lambert.jpg">
                                </div>
                            </div>

                            <div class="practices-exploration-section-right">
                                <div class="practices-exploration-image-left">
                                    <img src="http://2018.igem.org/wiki/images/7/7c/T--Cornell--Varner.jpg">
                                </div>
                                <div class="practices-exploration-text-right">
                                    <div class="practices-page-content-subheading">
                                        Prof. Jeffrey Varner, Robert Frederick Smith School of Chemical Engineering, Cornell University
                                        <div class="practices-explore-date">October 4, 2018</div>
                                    </div> 
                                    <p>Professor Varner’s research focuses on the development of physiochemical modeling tools. He is an expert on creating models for biological circuits, and therefore an invaluable resource for our project this year. In addition to confirming the foundations of our model, he spoke to us about parameter estimation, which is key to creating a model that accurately represents our system. Prof. Varner offered us suggestions on how to verify the parameters, including using cell-free analysis, analysis by individual colonies, and analysis by batch. Within our wet lab constraints, he recommended using <b>batch cultures as an average value for the output of our model</b>. He also introduced us to <b>Langevin modeling</b>, which offers the ability to incorporate noise as stochastic modeling does but without the complexity or long run-time. Beyond our model, he saw potential applications of our bio-circuit in drug delivery using cell-free protein-production implants.</p>
                                </div>
                            </div>
                        </div>
                        <div class="practices-implementation-wrapper">
                            <div class="standard-page-content-title">Implementation</div>
                            <hr class="yellow-accent-line-left">
                            <div class="standard-page-content-subheading"></div>
                            <ul>
                                <li>In our deterministic model, we included <b>first order degradation</b> for the repressor, <b>protease based degradation</b> of hrpR and hrpS, and <b>deactivating repressor degradation</b> of tetR and sigma-F.</li>
                                <li>We used measurements from our batch cell cultures to <b>estimate kinetic constants</b> for activator and repressor production, as well as the <b>time lags</b> involved in our biocircuit.</li>
                            </ul>
                        </div>
                    </div>
                    <!------------------------ PRACTICES TESTING ------------------------>
                    <div class="practices-text-wrapper" id="div4">
                        <div class="practices-recognition-wrapper">
                            <div class="standard-page-content-title">Recognition</div>
                            <hr class="yellow-accent-line-left">
                            <div class="standard-page-content-subheading"></div>
                            <div class="practices-recognition">
                                <p>Every system needs to be tested, especially one like ours with its numerous components. Throughout our research, we learned about testing and evaluation procedures, which influenced the way we analyzed our final system. We interviewed leading scientists in the fields of synthetic biology and circuitry to best understand how we could evaluate our circuit and present it to the scientific community.</p>
                            </div>
                        </div>
                        <div class="practices-exploration-wrapper">
                            <div class="standard-page-content-title">Exploration</div>
                            <hr class="yellow-accent-line-left">
                            <div class="standard-page-content-subheading"></div>
                            <div class="practices-exploration-section-left">
                                <div class="practices-exploration-text-left">
                                    <div class="practices-page-content-subheading">
                                        Prof. Guillaume Lambert, School of Applied and Engineering Physics, Cornell University
                                        <div class="practices-explore-date">April 9, 2018</div>
                                    </div> 
                                    <p>Part of Professor Lambert’s research focuses on the development and characterization of synthetic biological regulatory elements to guide the creation of biological circuits in cells and other active living systems. Hence, he was able to give us guidance into which experiments would be most valuable to convince someone that our circuit works. He suggested verifying that different frequency oscillations would lead to different amplitudes of output signal by <b>testing each component</b> of the high- and low-pass filter <b>separately</b> and demonstrating that our circuit is tunable.</p>
                                </div>
                                <div class="practices-exploration-image-right">
                                    <img src="http://2018.igem.org/wiki/images/4/41/T--Cornell--Lambert.jpg">
                                </div>
                            </div>

                            <div class="practices-exploration-section-right">
                                <div class="practices-exploration-image-left">
                                    <img src="http://2018.igem.org/wiki/images/f/f1/T--Cornell--Kan.jpg">
                                </div>
                                <div class="practices-exploration-text-right">
                                    <div class="practices-page-content-subheading">
                                        Prof. Edward Kan, School of Electrical and Computer Engineering, Cornell University 
                                        <div class="practices-explore-date">April 10, 2018</div>
                                    </div> 
                                    <p>Professor Kan’s research focuses on biosensors and electrical systems that interface with physiological systems. Since there are many aspects of our circuit to evaluate, this can make it difficult to identify which component is the most important. Professor Kan emphasized that we should focus on the power and time needed to activate our bio-circuit, incorporating knowledge about the overall <b>delay time</b>. During our refinement, we should focus on minimizing the energy needed to implement the circuit. Lower energy correlates to faster components. Further, we should use a prescribed test to test our circuit, essentially proving that a specified input signal leads to the right output.</p>
                                </div>
                            </div>

                            <div class="practices-exploration-section-left">
                                <div class="practices-exploration-text-left">
                                    <div class="practices-page-content-subheading">
                                        Prof. Guillaume Lambert, School of Applied and Engineering Physics, Cornell University 
                                        <div class="practices-explore-date">July 10, 2018</div>
                                    </div> 
                                    <p>As our project progressed from the ideation phase to the implementation phase, we discussed our developments with Professor Lambert once more. Overall, he was impressed with the biocircuit we developed. As we moved into the nuts and bolts of our testing phase, he urged us to carefully monitor the growth of our cells because the behavior of the circuit will change when cells reach their stationary growth phase. He recommended that we keep our cell concentration low so that the cells remain in the exponential growth phase. Flow cytometry would be a valuable tool for <b>real-time</b> and <b>rapid measurement</b> of the <b>fluorescence of individual cells</b>.</p>
                                </div>
                                <div class="practices-exploration-image-right">
                                    <img src="http://2018.igem.org/wiki/images/4/41/T--Cornell--Lambert.jpg">
                                </div>
                            </div>
                            
                            <div class="practices-exploration-section-right">
                                <div class="practices-exploration-image-left">
                                    <img src="http://2018.igem.org/wiki/images/9/93/T--Cornell--Swingle.jpg">
                                </div>
                                <div class="practices-exploration-text-right">
                                    <div class="practices-page-content-subheading">
                                        Prof. Bryan Swingle, School of Integrative Plant Science, Cornell University 
                                        <div class="practices-explore-date">August 6, 2018</div>
                                    </div> 
                                    <p>Professor Swingle is interested in developing methods to advance the genetic analysis of Pseudomonas syringae. This work has led him to create a recombination-mediated genetic engineering system for efficient manipulation of the P. syringae genome. With his expertise, he noted that the activity of hrpRS would ramp up quickly in our system. To provide tangible parameters in our testing, he suggested that we <b>set up reporter systems</b> to test the activity. Additionally, he recommended that we use an HRP inducing media and attach an RNA thermometer connected to a reporter to measure the time it takes to induce hrpRS.</p>
                                </div>
                            </div>
                        </div>
                        <div class="practices-implementation-wrapper">
                            <div class="standard-page-content-title">Implementation</div>
                            <hr class="yellow-accent-line-left">
                            <div class="standard-page-content-subheading"></div>
                            <ul>
                                <li>To verify that our circuit works, we <b>tested each component</b> of the high- and low- pass filter <b>separately</b>.</li>
                                <li>It is important to consider <b>delay time</b> and minimize the energy needed to implement the circuit to have faster components.</li>
                                <li>We used <b>fluorescence plate readers</b> for the <b>real-time</b> and <b>rapid measurement</b> of the <b>fluorescence of individual cells</b>.</li>
                                <li>To measure the time it takes to induce hrpRS, we <b>set up reporter systems</b> with the RNA thermometer connected to a reporter to provide tangible parameters in our testing.</li>
                            </ul>
                        </div>
                </div>
                <!------------------------ PRACTICES APPLICATIONS ------------------------>
                <div class="practices-text-wrapper" id="div5">
                    <div class="practices-recognition-wrapper">
                        <div class="standard-page-content-title">Understanding the Purpose of Our Project</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-recognition">
                            <p>When we first started our project back in March, we knew that there would be many applications for our project, but we weren’t sure where to start looking. We first turned to professors performing research in areas such as engineering physics and bioengineering to get their opinions on how to steer our project in a direction that would make the most significant impact on society. With the goal of creating something that could be useful not only in the lab, but also in our day-to-day lives, we then further delved into the possible applications with our team.</p>
                        </div>
                    </div>
                    <div class="practices-exploration-wrapper">
                        <div class="standard-page-content-title">Integrating Findings from Human Practices</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-exploration-section-left">
                            <div class="practices-exploration-text-left">
                                <p>When we interviewed Professor Lambert, he suggested we look towards co-culturing and how that might be used to regulate the microbiome. Co-cultures are difficult to maintain, but if each culture of bacteria responds to a different frequency, it would be possible to make some populations grow more while others grow less. This could apply to the human microbiome, increasing the concentration of beneficial bacteria to outnumber harmful ones. If the microbiome could respond to frequency, then perhaps it could also provide more information about our daily lives. Many people take pills, such as birth control, vitamins, or cholesterol medication, once a day. If we forgot to take our pill at the frequency of once a day, the microbiome could report to us and remind us when that frequency is off.</p>
                            </div>
                            <div class="practices-exploration-image-right practices-applications-icon">
                                <img src="http://2018.igem.org/wiki/images/2/23/T--Cornell--lambert.png">
                            </div>
                        </div>

                        <div class="practices-exploration-section-right">
                            <div class="practices-exploration-image-left practices-applications-icon">
                                <img src="http://2018.igem.org/wiki/images/a/ad/T--Cornell--march.png">
                            </div>
                            <div class="practices-exploration-text-right">
                                <p>When we interviewed Professor March, we discussed the possibility of using our system as a safety valve for ferritin nanoparticles so that they do not trigger during sustained heat or fluctuations of waves. Ferritin nanoparticles are normally used to present antigens or drugs to target cells in response to an external stimuli, such as heat. If these nanoparticles were to be placed in bacteria and then exposed to heat, it would cause them to express the antigens or drugs before they have reached their target cells. To address this concern, we would introduce the ferritin nanoparticles into the cells of frequency-sensitive bacteria. When we wish to have the ferritin nanoparticles carry out a specific task in the body, we would then use ultrasounds to heat up the bacteria to particular frequencies for selective expression. Though Professor March suggested applying this idea in environments with temperature fluctuations, he also challenged us to find a comprehensive application for our project, as our work should not simply exist in a vacuum.</p>
                            </div>
                        </div>

                        <div class="practices-exploration-section-left">
                            <div class="practices-exploration-text-left">
                                <p>When we interviewed Professor Varner, he introduced the prospect of cell-free synthesis, the formation of cellular products without the actual cells. In essence, we would take the cellular machinery involved in the production of, for example, proteins, and mix them together in a beaker with the frequency-sensitive bacteria. We would then add plasmids to the solution as an alternative to traditional cellular DNA and produce the desired reporters. The process would be modulated by the specific frequencies aimed toward the beaker solution. When the bacteria in the beaker pick up on the appropriate frequency, the desired products would be produced until the frequency is removed. During our discussion with Professor Varner, he mentioned the possibility implementing our biocircuit for cell-free synthesis implants for in vivo therapeutic production of proteins. In these, we could non-invasive control protein expression levels, allowing adaptability to dynamic patient needs. </p>
                            </div>
                            <div class="practices-exploration-image-right practices-applications-icon">
                                <img src="http://2018.igem.org/wiki/images/8/83/T--Cornell--varner.png">
                            </div>
                        </div>
                    </div>
                    <div class="practices-recognition-wrapper">
                        <div class="standard-page-content-title">Group Brainstorming</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-recognition">
                            <p>In addition to the feedback from the professors, our Policy & Practices sub-team held a large brainstorming session on September 15th during our weekly general body meeting where we opened the floor to our team members to collect a wide array of ideas for possible applications of our project. We received a huge response of suggestions ranging from food storage to applications in space. Each idea had potential, as well as their respective advantages and drawbacks, so it was difficult for us to settle on one single idea. However, we were eventually able to narrow our results down to five main categories: co-culturing, process control, biological clocks, medical devices, and cell therapy, three of which are discussed below. For each of our main ideas, we examined the specifics of the issue, as well as current solutions, if any, before identifying our approach and the industries in which they might be applicable.</p>
                        </div>
                    </div>
                    <div class="practices-exploration-wrapper">
                        <div class="practices-exploration-section-left">
                            <div class="practices-exploration-text-left">
                                <div class="practices-page-content-subheading" style="padding-bottom: 10px;">
                                    Application 1: Co-Culturing
                                </div> 
                                <p>For our first idea, we looked at the subject of co-culturing. Co-culturing happens when a a cell culture is set up with two or more different populations of cells growing with some degree of contact [4]. While this may be easy to set up, it is very difficult to maintain. The conditions of the culture have to be just right to support all the populations. They will compete for resources, but one population can not be allowed to overtake the others. One of the major reasons for setting up a co-culture is data acquisition, but even this needs to be carefully planned [4]. The dynamics of each individual population, as well as the interactions and the media, need to be studied in detail. Simple assays don’t exist for co-cultures. These problems of population maintenance, data collection, and culture conditions currently don’t have all-encompassing solutions [4]. Any more than two populations in a co-culture is too complex to maintain.</p><br><br>
                                <p>Oscillate could be part of the solution by introducing control over each population in the co-culture. Using different degradation tags and responsive promoters, it would be possible to tune the frequency at which Oscillate functions. Each population could respond to a different orthogonal frequency, which would then allow us to regulate each population’s growth. This means that one population could never overtake another, as the right frequency simply needs to be emitted to get the other population to grow faster. While Oscillate might not solve the issues with data collection, it would allow for co-culturing to become a more viable option.</p><br><br>
                                <p>This development could have significant implications for the microbiome and for biomanufacturing. The microbiome is a natural co-culture inside the human body. Many companies now advertise that they increase the population of “good” bacteria in the microbiome, but if each population were controlled with a different frequency, then the optimal ratio of populations could be maintained per person [6]. On the other hand, biomanufacturing utilizes multiple bioreactors to support biologically active systems, such as cultures [2]. Each bioreactor typically has one population where the product from one reactor is fed into another until the final product is created [2]. With Oscillate, there would be no need for multiple bioreactors since there would be the possibility of collapsing multiple bioreactors into one co-culture reactor. The product from one population feeds the next within the same reactor until the final product is produced.</p>
                            </div>
                            <div class="practices-exploration-image-right practices-applications-icon">
                                <img src="http://2018.igem.org/wiki/images/8/81/T--Cornell--co_culturing2.PNG">
                            </div>
                        </div>

                        <div class="practices-exploration-section-right">
                            <div class="practices-exploration-image-left practices-applications-icon">
                                <img src="http://2018.igem.org/wiki/images/7/78/T--Cornell--biological_clocks.png">
                            </div>
                            <div class="practices-exploration-text-right">
                                <div class="practices-page-content-subheading" style="padding-bottom: 10px;">
                                    Application 2: Biological Clocks
                                </div> 
                                <p>We also considered the idea of biological clocks. Biological clocks are innate mechanisms that control the cyclic activities of an organism [1]. One commonly discussed example of a biological clock is circadian rhythm, which is the 24-hour cycle of physiological processes in an organism. The main issue that arises with biological clocks is that when they become irregular, processes that should occur on a cycle are disrupted [1]. This negatively affects sleep cycles, digestion, body temperature, and other important functions that could lead to diseases such as diabetes, depression, and sleep disorders. The current solution is to treat the resulting symptoms with medication rather than the underlying causes. Medications are developed to help with sleep, to regulate insulin, or to produce antidepressant hormones. However, they never address the root cause: irregular biological clocks.</p><br><br>
                                <p>Using Oscillate, cells could sense when a cycle is irregular via frequency sensing and then respond accordingly. This would adjust the clock itself instead of simply treating the symptoms, thus reducing the need for additional medication and the risk of several disorders. One of the biggest impacts this could have is on sleep. Many people struggle with a regular sleep cycle, which could result in a lot of other problems [3]. With Oscillate, the hormones that promote sleep could be produced on a regular cycle and get people the sleep they need.</p>
                            </div>
                        </div>

                        <div class="practices-exploration-section-left">
                            <div class="practices-exploration-text-left">
                                <div class="practices-page-content-subheading" style="padding-bottom: 10px;">
                                    Application 3: Cell Therapy
                                </div> 
                                <p>Finally, the last topic we explored was cell therapy. Cell therapies use therapeutics at a cellular level in order to treat diseases [5]. Relevant uses today include stem cell therapy, cancer targeting, and hormonal therapy. Although there have been promising results with these therapies, there are also many issues, ranging from insufficient response to rejection to insufficient means of control [5]. Instead, simple dose-dependent administered drugs and other medicines are currently still being developed to combat these diseases.</p><br><br>
                                <p>Using Oscillate, frequency could be used to control the cells involved in these therapies. The frequency would penetrate the human tissue and offer more precise control over the therapies, such as where the cells are going and when they get activated. This could be extremely relevant in oncology, where T-cells have been modified to attack cancer but also end up attacking the healthy cells in the body. This would also be relevant in hormonal responses, where the cells would be able to sense the natural frequency of the hormones and then respond accordingly.</p>
                            </div>
                            <div class="practices-exploration-image-right practices-applications-icon">
                                <img src="http://2018.igem.org/wiki/images/2/2a/T--Cornell--cell_therapy.png">
                            </div>
                        </div>
                    </div>
                    <div class="practices-recognition-wrapper">
                        <div class="standard-page-content-title">Final Thoughts</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <div class="practices-recognition">
                            <p>Oscillate has many potential uses in many fields, ranging from medicine to microbiology to manufacturing. Our interviews with Professor Lambert, Professor March, Dr. Baig, and Professor Varner informed our initial ideas and inspired our creativity during brainstorming. Their expertise provided us with the ideal launch point for delving deeper into our project’s potential, and we found that the possibilities are endless.</p>
                        </div>
                    </div>
                    <div class="practices-implementation-wrapper">
                        <div class="standard-page-content-title">Works Cited</div>
                        <hr class="yellow-accent-line-left">
                        <div class="standard-page-content-subheading"></div>
                        <ol>
                            <li> Apte, C. V. (2012). Biological clocks: The coming of age. <i>International Journal of Applied and Basic Medical Research</i>, 2(1), 1.</li>
                            <li> Conner, J., Wuchterl, D., Lopez, M., Minshall, B., Prusti, R., Boclair, D., ... & Allen, C. (2014). The biomanufacturing of biotechnology products. In <i>Biotechnology Entrepreneurship</i>(pp. 351-385).</li>
                            <li> Dement, W. C., & Mitler, M. M. (1993). It's time to wake up to the importance of sleep disorders. <i>Jama, 269</i>(12), 1548-1550.</li>
                            <li> Goers, L., Freemont, P., & Polizzi, K. M. (2014). Co-culture systems and technologies: taking synthetic biology to the next level. <i>Journal of the Royal Society Interface, 11</i>(96), 20140065.</li>
                            <li> Mount, N. M., Ward, S. J., Kefalas, P., & Hyllner, J. (2015). Cell-based therapy technology classifications and translational challenges. <i>Phil. Trans. R. Soc. B, 370</i>(1680), 20150017.</li>
                            <li> Spector, T. (2018, January 02). How to boost your microbiome. Retrieved from https://www.sciencefocus.com/the-human-body/how-to-boost-your-microbiome/</li>
                        </ol>
                    </div>
                </div>
            </div>
        </div>
        <!------------------------ PRACTICES PAGE CONTENT END ------------------------>

        <!------------------------ FOOTER START ------------------------>
        <footer>
            <div class="footer-wrapper">
                <div class="icon-wrapper">
                    <a class="icon" href="http://facebook.com/cornelligem"><img src="http://2018.igem.org/wiki/images/3/30/T--Cornell--Facebook.jpg"></a>
                </div>
                <div class="icon-wrapper">
                    <a class="icon" href="http://twitter.com/cugem"><img src="http://2018.igem.org/wiki/images/0/02/T--Cornell--Twitter.jpg"></a>
                </div>
                <div class="icon-wrapper">
                    <a class="icon" href="https://www.instagram.com/cugem/"><img src="http://2018.igem.org/wiki/images/a/ae/T--Cornell--Instagram.jpg"></a>
                </div>
                <div class="icon-wrapper icon-wrapper-last">
                    <a class="icon" href="https://www.youtube.com/channel/UCJt-5JfyoucUJXC1EsTllhg"><img src="http://2018.igem.org/wiki/images/7/77/T--Cornell--Youtube.jpg"></a>
                </div>
            </div>
        </footer>
        <!------------------------ FOOTER END ------------------------>
    </div>
</body>
</html>